### PR TITLE
Profile readme

### PR DESCRIPTION
**Pull request by yliuuuu**
*Issue #, if available:* N/A

*Description of changes:*
Organization readme.  

[Rendered](https://github.com/yliuuuu/.github/blob/profile-readme/profile/README.md)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<p align="center">
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset="PartiQL_Logo_Dark.svg">
+        <source media="(prefers-color-scheme: light)" srcset="PartiQL_Logo_Light.svg">
+        <img src="PartiQL_Logo_Dark.svg" alt="PartiQL Logo" width="300" height="300">
+    </picture>
+</p>
+
+
+PartiQL is an expressive, SQL-compatible query language giving access to relational, semi-structured, and nested data. PartiQL is Open Sourced under PartiQL is open-sourced under the Apache 2.0 License.
```
Comment (rchowell) [inline profile/README.md]:
```suggestion
PartiQL is an expressive, SQL-compatible query language giving access to relational, semi-structured, and nested data. PartiQL is open sourced under the Apache 2.0 License. 
```

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        <img src="PartiQL_Logo_Dark.svg" alt="PartiQL Logo" width="300" height="300">
+    </picture>
+</p>
+
+
+PartiQL is an expressive, SQL-compatible query language giving access to relational, semi-structured, and nested data. PartiQL is Open Sourced under PartiQL is open-sourced under the Apache 2.0 License. 
+
+## Contributions
+
+* If you have questions or ideas towards PartiQL as a Language, check out our [GitHub Discussion](https://github.com/orgs/partiql/discussions) forum.
```
Comment (rchowell) [inline profile/README.md]:
```suggestion
* If you have questions or suggestions about the PartiQL Language, please check out our [GitHub Discussion](https://github.com/orgs/partiql/discussions) forum.
```

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+* Check out the [CONTRIBUTING.md](../CONTRIBUTING.md) file to learn how to contribute.
```
Comment (rchowell) [inline profile/README.md]:
```suggestion
* Please see the [CONTRIBUTING.md](../CONTRIBUTING.md) file to learn how to contribute.
```

**Review by rchowell — CHANGES_REQUESTED** [review]

File: profile/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<p align="center">
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset="PartiQL_Logo_Dark.svg">
+        <source media="(prefers-color-scheme: light)" srcset="PartiQL_Logo_Light.svg">
+        <img src="PartiQL_Logo_Dark.svg" alt="PartiQL Logo" width="300" height="300">
```
Comment (rchowell) [inline profile/README.md]:
```suggestion
        <img src="PartiQL_Logo_Dark.svg" alt="PartiQL Logo" width="100" height="100">
```

Just a preference, but it looks a bit big when rendered

**Review by rchowell — CHANGES_REQUESTED** [review]

**Review by rchowell — APPROVED** [review]

File: profile/README.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+<p align="center">
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset="PartiQL_Logo_Dark.svg">
+        <source media="(prefers-color-scheme: light)" srcset="PartiQL_Logo_Light.svg">
+        <img src="PartiQL_Logo_Dark.svg" alt="PartiQL Logo" width="100" height="100">
+    </picture>
+</p>
+
+
+PartiQL is an expressive, SQL-compatible query language giving access to relational, semi-structured, and nested data. PartiQL is open sourced under the Apache 2.0 License.
```
Comment (jpschorr) [inline profile/README.md]: PartiQL code, including reference implementations, is Apache 2.
But the language spec is Licensed a bit differently: https://github.com/partiql/partiql-lang/blob/main/LICENSE

Comment (yliuuuu) [inline profile/README.md]:
```suggestion
PartiQL is an expressive, SQL-compatible query language giving access to relational, semi-structured, and nested data. 

PartiQL code, including the reference implementations, is open sourced under the Apache 2.0 License. 
```

File: profile/README.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    </picture>
+</p>
+
+
+PartiQL is an expressive, SQL-compatible query language giving access to relational, semi-structured, and nested data. PartiQL is open sourced under the Apache 2.0 License. 
+
+## Contributions
+
+* If you have questions or suggestions about the PartiQL Language, please check out our [GitHub Discussion](https://github.com/orgs/partiql/discussions) forum.
+* Please see the [CONTRIBUTING.md](../CONTRIBUTING.md) file to learn how to contribute.
```
Comment (yliuuuu) [inline profile/README.md:15]:
```suggestion
* If you have questions or suggestions about the PartiQL Language, please check out our [GitHub Discussion](https://github.com/orgs/partiql/discussions) forum.
* If you want to contribute to the language specification, please check out our [RFC process](https://github.com/partiql/partiql-lang/blob/main/RFC-template.adoc). 
* Please see the [CONTRIBUTING.md](../CONTRIBUTING.md) file to learn how to contribute to our reference implementations.
```

File: profile/README.md (hunk 5 above)
Comment (yliuuuu) [reply profile/README.md]: How you like the purposed changes?

Comment (rchowell) [reply profile/README.md]: I recommend just leaving the last sentence out at the moment.

```suggestion
PartiQL is an expressive, SQL-compatible query language giving access to relational, semi-structured, and nested data.
```

**Pull request merged**